src/batched-json.js

```diff
@@ -17,6 +17,16 @@
 import {assertHttpsUrl} from './url';
 import {Services} from './services';
 import {getValueForExpr} from './json';
+import {user} from './log';
+
+/**
+ * @enum {number}
+ */
+export const UrlReplacementPolicy = {
```

Comment:
nit: Maybe `FetchUrlReplacementPolicy` since the policy does not apply to other cases?

Reply:
I think we don't need to disambiguate here since none/opt-in/all does fit the system in general. E.g. `<input>` and `<a>` tags currently require opt-in, and some URL attributes don't support substitutions at all.